src/App.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {Component} from 'react';
+import Navbar from './components/Navbar/Navbar';
+import UserBoard from './components/UserInput/UserBoard';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null
+    }
+  }
+
+  fetchData = () => {
+    axios.get("https://poetrydb.org/random")
+    .then((response) => {
+      console.log(response);
+      this.setState({
+        data: response.data[0]
+      })
+    }).catch((error) => {
+      console.log(error);
+    })
+  }
+
+  renderPoem = () => {
+    if (this.state.data) {
+      return (
+        <div>
+          <div>Title: {this.state.data.title}</div>
+          <div>Author: {this.state.data.author}</div>
+          <div>
+            {
+              this.state.data.lines.map((line) => {
+                return(<div>{line}</div>);
+              })
+            }
+          </div>
+        </div>
+      )
+    } else {
+      return(<div>Apologies, no poem available.</div>);
+    }
+  
+  }
+  
+  render () {
```

Comment:
there are a ton of different sections here! You could split them up into different components. And then you could make a state variable in App.js that keeps track of the current page and make a callback function that you pass to the navbar that changes that state variable. then you could conditionally render those components in app.js based on that variable. an example of that [implementation can be seen here](https://github.com/overlake333/debug)